generators/generator-microsoft-bot-adaptive/generators/app/templates/dotnet/webapp/Startup.cs

```diff
@@ -23,18 +23,21 @@ public void ConfigureServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-#pragma warning disable CA1801 // Review unused parameters
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
-#pragma warning restore CA1801 // Review unused parameters
         {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+        
             app.UseDefaultFiles();
-            app.UseStaticFiles();
```

Comment:
@gabog question for you on this one - I know we have used static files for things like skill manifests in the past. On looking at this again, I suspect we should keep UseStaticFiles despite it missing from the desired list of extensions in #618. Thoughts?

Reply:
Spoke offline - including.